alertwildfire_live_recording.py

```diff
@@ -158,7 +158,16 @@ def cleanup_archive(googleServices, dbManager, timethreshold):
             #does not yet change database
         time.sleep(Default_refresh_time)
     
-
+def test_System_response_time(trial_length = 10):
+    temporaryDir = tempfile.TemporaryDirectory()
+    trial = [x for x in range(0,trial_length)]
+    test_cameras = [camera['name'] for camera in alertwildfire_API.get_all_camera_info()[0:len(trial)]]
```

Comment:
I like that you're moving to testing multiple cameras, but couple of suggestions:
1) 'trial' is not very useful because all you're doing with it is taking the length, which is just trial_length.
2) It would be better to take cameras at random vs. always taking the first trial_length cameras.  So, I suggest
test_cameras - random.sample(alertwildfire_API.get_all_camera_info(), trial_length)